### DISCUSSION TITLE
Cleverio SS300 battery report null

### DISCUSSION THREAD

**Discussion by Jacp101** [Koenkk/zigbee2mqtt#20928] [Devices]
A couple of days ago I bought a cleverio SS300 humidity and temperature sensor. I get humidity and temperature readings in Z2MQTT, but the battery is not reporting. It says Null. I tried removing and readding the device, but no change. Do I need to do some sort of configuration to read the battery.

**Comment by panklev** [comment]
Have you been able to get this to work? I have the same problem and have tried the same as you without success. Still no battery information in payload..
![image](https://github.com/Koenkk/zigbee2mqtt/assets/105791573/3b58da57-cce4-4924-a007-4016520cf2d4)

**Reply by Jacp101** [reply]
Same issue still :-(

**Reply by Jacp101** [reply]
> Have you been able to get this to work? I have the same problem and have tried the same as you without success. Still no battery information in payload..
> 
> ![image](https://github.com/Koenkk/zigbee2mqtt/assets/105791573/3b58da57-cce4-4924-a007-4016520cf2d4)
> 
> 
> 
> 
> 
> 

Yesterday, I was able to get a barttery reading. Not sure what solved it, but I did the following:

I re-paired it this time directly to the adapter instead of through a different zigbee socket i had. Then tried the report trick(Z2M/reporting/ selected from dropdown and tried to send a request for remainingbattery), but recieved an error then, I immediately tried the cluster, and pressed refresh on the battery under exposes. Not sure which of the above did the trick, but i have a battery status now.
